Header header
uint16 sample_rate
uint16 chunk_size
string type
int16[] left_channel
int16[] right_channel
